gto-support-util/src/main/java/org/ccci/gto/android/common/util/content/Context.kt

```diff
@@ -4,16 +4,20 @@ import android.content.Context
 import android.content.res.Configuration
 import android.os.Build
 import android.os.LocaleList
-import org.ccci.gto.android.common.util.os.toTypedArray
+import org.ccci.gto.android.common.util.os.locales
 import java.util.Locale
 
+@OptIn(ExperimentalStdlibApi::class)
```

Comment:
This seems unfortunate.

Reply:
right now the `buildList` is experimental since it was just added to the language. it should be stable in Kotlin stdlib 1.4

Reply:
so technically the `buildList` function could change in new versions of Kotlin until it's stable, which is the reason it's requiring opt-in to use it

Reply:
I suppose you feel that it's stable enough to put into production?

Reply:
yeah, plus any changes to the method signature would cause compile errors when the stdlib is updated.

One of the benefits of dependabot is that we will know pretty quickly after a new version of stdlib is released if it breaks this code.